equinox-shipping/Domain/Container.fs

```diff
@@ -0,0 +1,49 @@
+module Container
+
+open Types
+open FSharp.UMX
+
+let [<Literal>] Category = "Container"
+
+module Events =
+
+    let streamName (containerId : string<containerId>) = FsCodec.StreamName.create Category (UMX.untag containerId)
```

Comment:
up to L7 too!

Reply:
Also, best to remove the `open FSharp.UMX` and make top level helpers in a `Domain.fs` or `Infrastructure.fs` like the other examples - that means a) we can Shift-F12 to find usages b) nobody needs to worry about this being UMX - UMX should be an impl detail only